src/Components/Milestones/Assessment.js

```diff
@@ -82,25 +48,35 @@ function Assessment() {
     <div className="assessment">
       <h6 className="contentTitle">Client Info</h6>
       <hr className="shortLine" />
-      <MilestoneClientInfo data={mockData} />
+      <MilestoneClientInfo data={jobseeker.clientInfo} />
       <h6 className="contentTitle">Education Info</h6>
       <hr className="shortLine" />
+      <MilestoneClientInfo data={mapArrays(jobseeker.degrees, 'Degree')} />
```

Comment:
degrees + certificates are a bit cramped

Reply:
added padding! Bc degrees & certificates are separate arrays, the padding b/w them vs the padding b/w each label for Client Info is slightly off :( but I'm blocked on how to fix that minor detail